docs/source/user_guide/installation/index.rst

```diff
@@ -50,8 +50,165 @@ Following GNU/Linux and macOS solvers are optionally available:
+Dependency                                                                       Minimum Version    pip extra          Notes
+================================================================================ ================== ================== ===================================================================================================================================================================================
+`sphinx <https://www.sphinx-doc.org/en/master/>`__                               1.5.0              docs               Sphinx makes it easy to create intelligent and beautiful documentation.
+`pydata-sphinx-theme <https://pydata-sphinx-theme.readthedocs.io/en/stable/>`__  -                  docs               A clean, Bootstrap-based Sphinx theme
+`sphinx_design <https://sphinx-design.readthedocs.io/en/latest/>`__              -                  docs               A sphinx extension for designing
+`sphinx-copybutton <https://sphinx-copybutton.readthedocs.io/en/latest/>`__      -                  docs               To copy codeblocks
+`myst-parser <https://myst-parser.readthedocs.io/en/latest/>`__                  -                  docs               For technical & scientific documentation.
+`sphinx-inline-tabs <https://sphinx-inline-tabs.readthedocs.io/en/latest/>`__    -                  docs               Add inline tabbed content to your Sphinx documentation.
+================================================================================ ================== ================== ===================================================================================================================================================================================
+
+.. _install.examples_dependencies:
+
+Examples dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[examples]"``
+
+================================================================================ ================== ================== ===================================================================================================================================================================================
+Dependency                                                                       Minimum Version    pip extra          Notes
+================================================================================ ================== ================== ===================================================================================================================================================================================
+`pre-commit <https://pre-commit.com/index.html>`__                               -                  examples           For managing and maintaining multi-language pre-commit hooks.
```

Comment:
I think pre-commit and black should only be a dev dependency

Reply:
Sure, they're now being installed with `[dev]` only.